connector/clients/kutis_client.py

```
@@ -0,0 +1,96 @@
+            async with self._session.get(
+                loginRedirectUrl,
+                allow_redirects=False,
+            ) as ssoResponse:
+                if ssoResponse.status != 302:
+                    raise ClientException(errorType=ErrorType.KUTIS_ERROR)
+
+                ssoRedirectUrl = ssoResponse.headers.get("Location")
+                if not ssoRedirectUrl:
+                    raise ClientException(errorType=ErrorType.KUTIS_ERROR)
+
+            async with self.kutisSession.get(
```

Pylint (3.3.7)

[error] Instance of 'KutisClient' has no 'kutisSession' member

(E1101)